vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -157,7 +157,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("100m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("100Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		klog.InfoS("d", "d", d)
```

Comment:
Let's find out what this line is meant for and give it a more useful text. I'm not entirely sure this line should actually be there, I thought tests should log with something like t.Log() instead anyways.

Reply:
Good catch. I've updated the code to use `framework.Logf`, similar to how it's implemented here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/replica_set.go#L643.